crates/ark/src/modules/rstudio/stubs.R

```diff
@@ -36,3 +36,27 @@
     path <- normalizePath(path)
     invisible(.ps.ui.openWorkspace(path, newSession))
 }
+
+#' @export
+.rs.api.viewer <- function (url, height = NULL) {
+    if (!is.character(url) || (length(url) != 1))
+        stop("url must be a single element character vector.")
+    if (identical(height, "maximize"))
+        height <- -1
+    if (!is.null(height) && (!is.numeric(height) || (length(height) !=
+        1)))
+        stop("height must be a single element numeric vector or 'maximize'.")
+    if (is.null(height)) {
+        height <- 0
+    }
```

Comment:
This is what i was looking for in the other two places `height` is used! Could be a little helper like `height <- check_viewer_height(height)`

Reply:
done!